server/mailer/index.js

```diff
@@ -65,12 +71,17 @@ const sendSignUpConfirmationLink = (req, resp) => {
 };
 
 const sendResetPasswordLink = (req, resp) => {
-  const { email: receiver, displayName, resetToken } = resp.locales;
+  const {
+    email: receiver,
+    expirationDate,
+    displayName,
+    resetToken
+  } = resp.locales;
   const host = req.get('host');
+  const expirationTime = new Date(expirationDate).toLocaleString();
   const resetUrl = `${host}/auth/recovery-password/${resetToken}`;
   const title = `${PROJECT_NAME} - Reset your password`;
-  const info =
-    'Reset your password by clicking reset button. If you have not requested password reset to your account, just ignore this message.';
+  const info = `Reset your password by clicking reset button. If you have not requested password reset to your account, just ignore this message. <b>Remember that reset button will be only active until ${expirationTime}.</b> `;
```

Comment:
proposal: what do you think about displaying not only expiration date but time duration also (1 hour,  2 days)?

Reply:
It's already there. The user gets full date and time info in the email message.

![_EOC-493__Send_information_how_long_tokens_are_valid_by_adamklepacz_·_Pull_Request__359_·_sandstreamdev_eoc](https://user-images.githubusercontent.com/27632432/65755634-cc510200-e113-11e9-84b9-da52688580ca.png)

Reply:
Please check emails from other sevices and try to reformat ours as these has wall of text without any paragraphs which is hard to read.
Also tell user for how long the link will be valid e.g. 3 hours, so that user do not have to make some math how much time he/she got until link will expire.